playbooks/juju/pre.yaml

```diff
@@ -56,3 +56,16 @@
           --model-default=/tmp/charm-test-infra/juju-configs/model-default-serverstack.yaml \
           --config=/tmp/charm-test-infra/juju-configs/controller-default.yaml \
           {{ serverstack_cloud.region_name }}/{{ serverstack_cloud.region_name }}
+    - name: 'Configure cloudinit-userdata model-default'
+      shell:
+        cmd: |
```

Comment:
are you sure about `cmd: |`? this is not the usual way I think. Usually we do multiline shell scripts like this: https://github.com/openstack-charmers/zosci-config/blob/master/playbooks/juju/cleanup.yaml#L6

Reply:
Actually this should work: https://github.com/ansible/ansible/issues/39137#issuecomment-383453939

Reply:
Used this syntax based on https://github.com/ansible/ansible/issues/39137#issuecomment-383453939